frontend/src/scenes/notebooks/Nodes/components/NotebookNodeTitle.tsx

```diff
@@ -0,0 +1,63 @@
+import { KeyboardEvent } from 'react'
+import { useActions, useValues } from 'kea'
+import { notebookNodeLogic } from '../notebookNodeLogic'
+import { useEffect, useState } from 'react'
+import { LemonInput, Tooltip } from '@posthog/lemon-ui'
+import { notebookLogic } from 'scenes/notebooks/Notebook/notebookLogic'
+
+export function NotebookNodeTitle(): JSX.Element {
+    const { isEditable } = useValues(notebookLogic)
+    const { nodeAttributes, title, titlePlaceholder } = useValues(notebookNodeLogic)
+    const { updateAttributes } = useActions(notebookNodeLogic)
+    const [editing, setEditing] = useState(false)
+    const [newValue, setNewValue] = useState('')
+
+    useEffect(() => {
+        setNewValue(nodeAttributes.title ?? '')
```

Comment:
This confused me at first. Could we not just set the value initially:

```
const [newValue, setNewValue] = useState(nodeAttributes.title ?? '')
```

Reply:
Oh I suppose you'd have to do
```
setNewValue(nodeAttributes.title ?? '')
```
if someone cancels editing. So probably as easy to have the useEffect

Reply:
yeah its exactly for the editing state change